samples/bluetooth/central_hr/src/main.c

```diff
@@ -207,7 +177,7 @@ static void device_found(const bt_addr_le_t *addr, s8_t rssi, u8_t type,
 	/* We're only interested in connectable events */
 	if (type == BT_LE_ADV_IND || type == BT_LE_ADV_DIRECT_IND) {
 		/* TODO: Move this to a place it can be shared */
```

Comment:
I guess we can remove the TODO comment as well.

Reply:
Good point.